README.md

```diff
@@ -59,4 +47,48 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
 
-See the section about [deployment](https://create-react-app.dev/docs/deployment/) for more information.
+See the section about [building and deploying](https://vitejs.dev/guide/build) for more information.
+
+
+## Usage
+
+The REMS SMART on FHIR app interacts with the [REMS Admin](https://github.com/mcode/rems-admin), an [EHR](https://github.com/mcode/test-ehr), and the [Pharmacy Information Management System](https://github.com/mcode/pims).  These apps are provided as part of the REMS ecosystem, but any individual part may be swapped out for something custom.  The EHR in particular can be easily switched by launching the SMART app from a different EHR.  Launching the app can be easily done through the [Request Generator](https://github.com/mcode/request-generator), a front end app for EHRs that sends and recieves CDS Hooks.  The SMART app must be launched through an EHR or through the Request Generator.
```

Comment:
Does the SMART on FHIR app still interact with PIMS? Checking the pharmacy status involves a check to the EHR FHIR server.

Reply:
It does not, the env var was deleted; the src/views/Patient/MedReqDropDown/MedReqDropDown.tsx and src/views/Patient/MedReqDropDown/pharmacyStatus/PharmacyStatus.tsx use the response from test-ehr's MedicationRequest endpoint